pom.xml

```diff
@@ -123,11 +123,11 @@
       --add-opens=java.base/java.util=ALL-UNNAMED
       --add-opens=java.base/java.util.concurrent=ALL-UNNAMED
       --add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED
+      --add-opens=java.base/jdk.internal.misc=ALL-UNNAMED
```

Comment:
Is this command line OK for Java 11?

Reply:
It's OK, we have CI for JDK 11

Reply:
the `--add-opens` flag was introduced since JDK 9, see https://docs.oracle.com/javase/9/migrate/toc.htm#JSMIG-GUID-12F945EB-71D6-46AF-8C3D-D354FD0B1781 for details.